apis/extension/slo_controller_config.go

```diff
@@ -34,6 +34,13 @@ const (
 	SystemConfigKey            = "system-config"
 )
 
+// +k8s:deepcopy-gen=true
+type NodeCfgProfile struct {
```

Comment:
how about add some ut as yaml format for compatibility checking, to make sure that old format configmap yaml can be parsed to new struct.

Reply:
ut of colocation_cm_event_handler_test have case for old config json to new struct